src/tracer.cc

```diff
@@ -560,6 +562,9 @@ TraceManager::Trace::StartSpan(
     if (std::string(request_id) != "") {
       span->SetAttribute("triton.request_id", request_id);
     }
+    triton::common::TritonJson::WriteBuffer buffer;
+    PrepareTraceContext(span, &buffer);
```

Comment:
I would like users to be more clear in what they want. Since by default trace mode is `triton` on front side, I think it is reasonable to have this requirement on the python_be side as well

Reply:
Since in this stage mode is not exposed to python_be, I think it is reasonable to get rid of `mode` argument in `get_context`. It will still return `None` if passed context is an empty string

Reply:
I prefer not having the `mode` for now if not needed, since it seems like the user can choose the mode, but the mode is already set on the server side.

Reply:
refactored